repobee_sanitizer/_sanitize.py

```diff
@@ -13,6 +13,12 @@
 REPLACE_WITH = "REPOBEE-SANITIZER-REPLACE-WITH"
 END_BLOCK = "REPOBEE-SANITIZER-END"
 
+SANITIZER_MARKERS = (
+    START_BLOCK,
+    REPLACE_WITH,
+    END_BLOCK,
+)
```

Comment:
Although not 100% relevant to this PR, I approve of this change. It's for the better.